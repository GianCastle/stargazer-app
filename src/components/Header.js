import React from 'react'

export default function Header() {
  return (

    <div className='app__wrapper' data-testid="header">
      <nav className="navbar navbar-dark bg-dark">
        <div className="navbar-brand">Stargazer search</div>
      </nav>
    </div>

  )
}

