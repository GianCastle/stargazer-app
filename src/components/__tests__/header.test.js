import { render } from "@testing-library/react";
import Header from "../Header";


describe('header component', () => {
  it('should render', () => {
    const { wrapper, getByTestId } = render(<Header />);
    const header = getByTestId('header');

    expect(header).toHaveTextContent('Stargazer search');
    expect(wrapper).toMatchSnapshot();
  });
})