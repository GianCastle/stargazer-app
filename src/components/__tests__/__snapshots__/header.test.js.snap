// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component should render 1`] = `undefined`;
